editor/js/ui/sidebar.js

```diff
@@ -30,7 +30,7 @@ RED.sidebar = (function() {
     function addTab(title,content,closeable) {
         $("#sidebar-content").append(content);
         $(content).hide();
-        sidebar_tabs.addTab({id:"tab-"+title,label:title,closeable:closeable});
+        sidebar_tabs.addTab({id:"tab-"+title,label:RED._("sidebar."+title),closeable:closeable});
```

Comment:
Any node could add a sidebar tab, so the title of the tab won't be in the editors message catalog as this change assumes. Callers to `addTab` will have to retrieve the NLS'd title for themselves and pass it in here.